ls/ls.go

```diff
@@ -56,3 +60,19 @@ func isHiddenPath(path string, forceHidden bool) bool {
 	}
 	return path[0] == dotCharacter
 }
+
+func (l *LS) ShowDirStructure() error {
+    file, err := os.Stat(l.Dir)
+	if err != nil {
+        return err
+    }
+ 
+	if file.IsDir() {
+		fmt.Fprintln(l.StdOut, file.Name()+"/")
+	}else{
+		fmt.Println(file.Name())
```

Comment:
use `fmt.Fprintln(l.StdOut, file.Name())`

Reply:
Done correcting

Reply:
LGTM. Keep up the good work